kolibri/utils/cli.py

```diff
@@ -456,26 +456,37 @@ def configure():
     pass
 
 
+def _format_env_var(envvar, value):
+    if value.get("deprecated", False) or envvar in value.get(
+        "deprecated_envvars", tuple()
+    ):
+        return click.style(
+            "{envvar} - DEPRECATED - {description}\n\n".format(
+                envvar=envvar, description=value.get("description", "")
+            ),
+            fg="yellow",
+        )
+    return "{envvar} - {description}\n\n".format(
+        envvar=envvar, description=value.get("description", "")
+    )
```

Comment:
maybe one day we'll get to use fstrings!

slightly more concise:

```suggestion
    return "{} - {}\n\n".format(envvar, value.get("description", ""))
```

Reply:
I find named template variables easier to grok, personally.